### DISCUSSION TITLE
Netty File transfer

### DISCUSSION THREAD

**Discussion by veena-g123** [netty/netty#11991] [Q&A] [answered]
how can i download/send  a zip file [which is in the server] to client using netty through opened port[which connects server and client]?i am using java and also where can i get a clear picture on netty because i am new to it
### Netty version
4.149

**Comment by amizurov** [comment, marked as answer]
Hi, you can see an example here: https://github.com/netty/netty/tree/4.1/example/src/main/java/io/netty/example/file or if you are interested in web example: https://github.com/netty/netty/tree/4.1/example/src/main/java/io/netty/example/http/file

**Reply by zcz3313** [reply]
@amizurov  what does  code example of client side would be？
